app/_locales/es/messages.json

```diff
@@ -2472,7 +2472,7 @@
     "description": "The description for the `snap_confirm` permission"
   },
   "permission_ethereumAccounts": {
-    "message": "Ver las direcciones de las cuentas permitidas (requerido)",
+    "message": "Ver las direcciones, balances, actividad de las cuentas permitidas y sugerir transacciones para aprobar (requerido)",
```

Comment:
I would change `balances` for `saldos` - using the Spanish translation would be more consolidated as it is already used in the rest of the Extension